yt/visualization/tests/test_line_plots.py

```diff
@@ -34,48 +32,62 @@ def image_from_plot(filename_prefix):
     test.prefix = test_prefix
     return test
 
-tri2 = "SecondOrderTris/RZ_p_no_parts_do_nothing_bcs_cone_out.e"
-
-@requires_ds(tri2)
```

Comment:
Can giving this a generated dataset be done in a separate PR? It seems orthogonal.

Reply:
These tests modifications are included in this PR, just to test whether this PR works or not? If I change these tests in another PR, then testing this PR would be quite difficult.